examples/analyze.py

```diff
@@ -39,38 +39,49 @@
              }
 
 if __name__ == "__main__":
-    # Check that directories exists, create if it doesn't.
+    # Check that directories exists, create if they don't.
     if not os.path.exists(beads_plot_dir):
         os.makedirs(beads_plot_dir)
     if not os.path.exists(gated_plot_dir):
         os.makedirs(gated_plot_dir)
 
-    # Process beads data
+    # Load beads data
     print("\nProcessing beads...")
     beads_data = FlowCal.io.FCSData('{}/{}'.format(directory, beads_file))
     print("Beads file contains {} events.".format(beads_data.shape[0]))
 
-    # Trim
+    # Transform to Relative Fluorescence Units (rfi), commonly known as
+    # arbitrary units (a.u.)
+    beads_data = FlowCal.transform.to_rfi(beads_data,
+                                          sc_channels + fl_channels)
+
+    # Gating
+
+    # Remove first and last events. Transients in fluidics can make the
+    # first few and last events slightly different from the rest.
     beads_data = FlowCal.gate.start_end(beads_data, num_start=250, num_end=100)
+
+    # Remove saturating events in forward/side scatter.
     beads_data = FlowCal.gate.high_low(beads_data, sc_channels)
 
-    # Density gate
+    # Apply density gating
     print("\nRunning density gate on beads data...")
     gated_beads_data, __, gate_contour = FlowCal.gate.density2d(
         data=beads_data,
         channels=sc_channels,
         gate_fraction=0.3,
         full_output=True)
-    # Plot
+
+    # Plot forward/side scatter density plot and fluorescence histograms
     plt.figure(figsize = (6,4))
     FlowCal.plot.density_and_hist(
         beads_data,
         gated_beads_data, 
         density_channels=sc_channels,
         hist_channels=fl_channels,
         gate_contour=gate_contour, 
-        density_params={'mode': 'scatter'}, 
-        hist_params={'ylim': (0, 1000)},
+        density_params={'mode': 'scatter', 'xscale': 'log', 'yscale': 'log'},
+        hist_params={'ylim': (0, 1000), 'xscale': 'log'},
```

Comment:
did you go with `log` over `logicle` for a reason? again, figured we'd do `logicle` everywhere to be consistent. (same for all plots in this file)